version1/02 - Object types.md

```diff
@@ -258,29 +258,31 @@ The following notation is used in the tables below:
+| M601      | string    | opprettetAv                   |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |      |        |                                        |
+| M601      | string    | opprettetAvBrukerIdent        |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |      |        |                                        |
+| M010      | string    | sakspartIdent                 |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M302      | string    | sakspartNavn                  |    x     |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M303      | string    | sakspartRolle                 |    x     |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+|           | string    | foedselsnummer                |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+|           | string    | dnummer                       |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M406      | string    | postadresse                   |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M407      | string    | postnummer                    |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M408      | string    | poststed                      |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M409      | string    | land                          |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M410      | string    | epostadresse                  |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M411      | string    | telefonnummer                 |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+| M412      | string    | kontaktperson                 |          |             |           |  |  x   |  x   |   x   |  x   |      |        |                                        |
+|           |           |                               |          |             |           |  |      |      |       |      |      |        |                                        |
+|           | object    | virksomhetsspesifikkeMetadata |          |             |           |  |  x   |  x   |       |      |      |        |                                        |
```

Comment:
Field no. should be M711 here, for Sakspart only, as it is defined in Noark 5.5.

Reply:
Added in 58c9093.